client/package.json

```diff
@@ -42,51 +42,49 @@
     "core-js": "^2.5.4",
     "fecha": "^4.2.3",
     "file-saver": "2.0.5",
-    "hammerjs": "^2.0.8",
     "material-design-icons": "^3.0.1",
     "moment": "^2.29.4",
-    "ng2-charts": "2.2.3",
-    "ngx-color-picker": "11.0.0",
-    "ngx-toastr": "11.3.3",
-    "panzoom": "^9.4.2",
-    "pdfmake": "^0.2.5",
+    "ng2-charts": "^2.4.3",
+    "ngx-color-picker": "10.1.0",
+    "ngx-toastr": "13.1.0",
+    "panzoom": "^9.4.3",
+    "pdfmake": "^0.2.6",
     "rxjs": "~6.5.4",
     "socket.io-client": "^2.4.0",
-    "tslib": "^1.10.0",
+    "tslib": "^2.0.0",
     "uplot": "1.6.15",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
```

Comment:
Hi, maybe you know what I'm missing?

...\FUXA\client> ng lint --fix

Linting "FUXA"...
An unhandled exception occurred: Failed to load plugin '@angular-eslint' declared in '.eslintrc.json#overrides[0]': Class extends value undefined is not a constructor or nullReferenced from: C:\work\FUXA\pull-request\FUXA-PR-573\FUXA\client\.eslintrc.json      
See "C:\Users\defau\AppData\Local\Temp\ng-b8op7o\angular-errors.log" for further details.

Reply:
Hi, sorry, missed to check it, I'll research the issue tomorrow. I think it's incompatible with eslint 8

Reply:
Fixed by downgrading `eslint` package version